reference/orders.v2.oas2.yml

```diff
@@ -3915,6 +3918,10 @@ components:
           description: Override value for the total, including tax. If specified, the field `total_ex_tax` is also required. (Float, Float-As-String, Integer)
           example: '225.0000'
           type: string
+        is_tax_inclusive_pricing:
```

Comment:
seems like we don't need it here, otherwise it will appear in the `POST /v2/orders` body:

![post_body](https://github.com/user-attachments/assets/23dfedc0-fab9-4a61-81e9-98714dc3c5f7)